crates/core/types/block.rs

```diff
@@ -334,6 +324,28 @@ fn check_gas_limit(gas_limit: u64, parent_gas_limit: u64) -> bool {
         && gas_limit >= GAS_LIMIT_MINIMUM
 }
 
+// Calculates the base fee per blob gas for the current block based on it's parent excess blob gas
```

Comment:
init: t's weird that all these helper functions are here (not only the ones you added)

Reply:
I agree. Maybe we need a `core::utils` module to hold this logic?

Reply:
I wouldnt use a generic utils module. Maybe gas_helpers or 4844_helpers? We can also leave them here for now until we think of a better place

Reply:
I have created #324  for this.